catalog/nato-app/src/main/resources/features.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+
+<!--
+     NOTE: This features XML file uses the Karaf "install" attribute to specify whether a feature
+     is automatically installed when an app's KAR file is hot deployed.
+
+     Description of the install attribute from the Karaf features XSD:
+
+     "Installation mode. Can be either manual or auto. Specifies whether the feature should be
+     automatically installed when dropped inside the deploy folder. Note: This attribute doesn't
+     affect feature descriptors that are installed from the command line or as part of the
+     org.apache.karaf.features.cfg."
+-->
+<features name="${project.artifactId}-${project.version}"
+          xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
+
+    <feature name="catalog-stanag4559-source" install="manual" version="${project.version}"
+             description="DDF Catalog STANAG 4559 Source">
+        <bundle>mvn:alliance.catalog.nato.stanag4559/catalog-stanag4559-source/${project.version}</bundle>
+    </feature>
+
+    <feature name="nato-app" install="auto" version="${project.version}"
+             description="The DDF Nato App ::DDF Nato">
+        <feature>catalog-stanag4559-source</feature>
```

Comment:
Added the pre-req for `catalog-app` in https://github.com/codice/alliance/pull/3